plottable.js

```diff
@@ -8993,6 +8993,7 @@ var Plottable;
                 _super.call(this);
                 this._interpolator = "linear";
                 this._autorangeSmooth = false;
+                this._croppedRendering = false;
```

Comment:
really minor, but we usually have the flag name to be consistent with the function name